### PR TITLE
Issue#1: Add live scroll counter to the right corner of the page

### LINKED ISSUE #1: Display live scroll distance pedometer
Instead of displaying the scroll distance in the popup menu, let's show it live in the corner of the website.

### PR DESCRIPTION
**Pull request by tin0312**
fixes #1

### CODE REVIEW

**Comment by tin0312** [comment]
fixed #1

File: content.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.fps-counter {
```
Comment (DukeManh) [inline content.css]:
```suggestion
.scroll-counter {
```

File: content.js [hunk 2]
```diff
@@ -6,14 +6,20 @@ let lastScrollPosition = initialScrollPosition;
...
 
 const updateScrollDistance = () => {
   const scrollPosition = window.scrollY;
 
   scrollDistance += Math.abs(scrollPosition - lastScrollPosition);
   lastScrollPosition = scrollPosition;
+
+  // Make the scroll distance display live in the fps-counter
+  const distanceCM = pixelToCm(scrollDistance).toFixed(0);
+  div.innerHTML = `Scroll Distance : ${distanceCM} cm `;
```
Comment (DukeManh) [inline content.js]:
```suggestion
  `${distanceCM} cm`;
```

**Review by DukeManh — CHANGES_REQUESTED** [review]

**Review by DukeManh — COMMENTED** [review]
It's very close

File: content.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    position: fixed;
+    top: 0;
+    right: 0;
+    width: 200px;
+    height: 100px;
+    color: green;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (DukeManh) [inline content.css]: Add `z-index` here to make sure the counter is always visible

**Review by DukeManh — APPROVED** [review]

**Pull request merged**